ebu_tt_live/examples/config/producer_resequencer_consumer_fs.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Seems like for this PR we want this same thing with a deduplicator node between the resequencer and the consumer.